lib/BlueSCSI_platform_RP2040/scsiPhy.cpp

```diff
@@ -1,5 +1,8 @@
 // Implements the low level interface to SCSI bus
 // Partially derived from scsiPhy.c from SCSI2SD-V6
+//
+// Copyright (c) 2022 Rabbit Hole Computing™
+// Copyright (c) 2014 Michael McMaster
```

Comment:
This notice does not match your notice. https://github.com/ZuluSCSI/ZuluSCSI-firmware/blob/main/lib/ZuluSCSI_platform_RP2040/scsiHostPhy.cpp

Reply:
Hmm, one of those is "scsiPhy.cpp" which is derived from SCSI2SD, as noted also in [ZuluSCSI version of the file](https://github.com/ZuluSCSI/ZuluSCSI-firmware/blob/main/lib/ZuluSCSI_platform_RP2040/scsiPhy.cpp). The one in your link is "scsiHostPhy.cpp" for the host/initiator mode that is independent implementation. I think the copyright comments here are correct?

Reply:
You are correct, I was looking at host vs non.